test/integration/default/inspec/default_spec.rb

```diff
@@ -60,6 +60,13 @@
     its('content') { should match(%r{nexus-context-path=/}) }
   end
 
+  describe user('nexusbar') do
+    it { should exist }
+    its('uid') { should eq 1234 }
+    its('group') { should eq 'nexusbar' }
+    its('gid') { should eq 4568 }
```

Comment:
You put `5678` in the test recipe (reason of the failing test)

Reply:
Thanks, was a bit quick on typing the id :D